test/pre-post-extension.js

```diff
@@ -0,0 +1,56 @@
+import chai, {expect} from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import Stipulate from '../src';
+
+chai.use(chaiAsPromised);
+
+class ExtStip extends Stipulate {
```

Comment:
Do you have a name in mind? I think it's fine as is, but I'll change it if you've got one you want.